src/mongo/db/repl/elect_cmd_runner.cpp

```diff
@@ -114,7 +114,8 @@ namespace repl {
 
         if (response.isOK()) {
             BSONObj res = response.getValue().data;
-            LOG(1) << "elect res: " << res.toString();
+            log() << "received " << res["vote"] << " votes from " << request.target;
```

Comment:
Please remove "votes" or only print the value of the vote field (not including the field name). 

This looks odd as is:  "received vote: 1 votes from ip-10-136-108-244:27017"